template/src/index.js

```diff
@@ -0,0 +1,11 @@
+import ReactDOM from 'react-dom';
+
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```

Comment:
I assume this is a CRA feature, right? Would there be a benefit in using our own [signal](https://github.com/sumup-oss/signal) library instead?

Reply:
Yes, this is part of CRA. I think it would be cool to use signal. Would there be any restrictions on how that _can_ be used and how it needs to be configured?

Reply:
Hm, yes, the [configuration](https://github.com/sumup-oss/signal#installation) is more involved than I thought. 

TL;DR: You need to generate your own script and inline it.